src/vm/codeman.cpp

```diff
@@ -1439,6 +1439,8 @@ static void LoadAndInitializeJIT(LPCWSTR pwzJitName, OUT HINSTANCE* phJit, OUT I
 
         EX_TRY
         {
+            bool fContinueToLoadJIT = false;
+#if !defined(FEATURE_CORECLR)
```

Comment:
This is being done to ensure consistency for CoreCLR based JIT contract.

> So I assume the JIT will be built with SELF_NO_HOST defined for all cases of FEATURE_CORECLR defined

This is already true in the open.